src/Service/MDS/IMetaDataService.cs

```diff
@@ -7,4 +7,11 @@ public interface IMetaDataService
     Task<IEnumerable<string>> GetAttestationTypesAsync();
     Task<IEnumerable<EntryResponse>> GetEntriesAsync(EntriesRequest request);
     Task<IEnumerable<string>> GetCertificationStatusesAsync();
+
+    /// <summary>
+    /// Checks whether all the given authenticators exist in the FIDO2 MDS.
+    /// </summary>
+    /// <param name="aaGuids"></param>
+    /// <returns>Returns `true` if all the authenticators exist in the FIDO2 MDS; otherwise, `false`.</returns>
+    Task<bool> IsExistsAsync(IReadOnlyCollection<Guid> aaGuids);
```

Comment:
@jonashendrickx I think the naming here could just be `ExistsAsync`, as that seems to be what [dotnet uses](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.exists?view=net-8.0)?

Reply:
@abergs  renamed